imap/itip_support.c

```diff
@@ -714,6 +714,21 @@ static int deliver_merge_request(const char *attendee,
         icalcomponent_add_component(ical, new_comp);
     }
 
+    /* Remove components of old object that still remain in the hash table */
```

Comment:
This is the case of "Remove components of old object that are not present in the new object"? If so, the comment should just tell so, I had to read the code to figure out what's the purpose of the hash table.

Reply:
Added clarifying comment